Reader: Update the styling of the counters

evilluendas opened the pull request:
#### Changes proposed in this Pull Request
 
* Small tweaks to the counters recently introduced in the Reader: if the site's name is very long, they could overlap. These changes keep them separated; they also make the counters a bit taller and adjust the colors so they work better with the different themes.

**Before**
![image](https://user-images.githubusercontent.com/820374/88195668-33f2b380-cc38-11ea-9689-37b9d26833cb.png)

**After**
![image](https://user-images.githubusercontent.com/820374/88195711-3c4aee80-cc38-11ea-8179-ac294023c44b.png)


#### Testing instructions

* Switch to this PR and go to /read
* Does everything look fine in the sidebar? Is the layout breaking in any way?

blowery commented:
Styling looks broken when the site name gets clipped and the count is two digits

<img width="247" alt="Screen Shot 2020-07-22 at 11 50 04 AM" src="https://user-images.githubusercontent.com/14350/88198293-8ec4e480-cc11-11ea-9388-c36acd0f23b1.png">

evilluendas commented:
Great catch, thanks! I think I've fixed it, could you check again?

![image](https://user-images.githubusercontent.com/820374/88202221-77e9b680-cc40-11ea-9cad-58094e3b81fc.png)

blowery commented:
It's better! Only oddity I see now is that bubble is a different width for `1` than for other single digit numbers.

<img width="247" alt="Screen Shot 2020-07-22 at 12 39 09 PM" src="https://user-images.githubusercontent.com/14350/88203835-6ab8d180-cc18-11ea-9fa2-d415dbd8ef38.png">

evilluendas commented:
Fixed! It had min-width before but I removed it because it was behaving weirdly — turns out it was the flex shrink:

![image](https://user-images.githubusercontent.com/820374/88205237-5db1d780-cc44-11ea-938f-c279f4724fbc.png)
